app/views/usage/index.html.erb

```diff
@@ -0,0 +1,87 @@
+<div class="row">
+  <div class="col-md-12">
+    <h1>Usage statistics</h1>
+  </div>
+</div>
+<div class="row">
+  <div class="col-md-12">
+    <h3 class="pull-right"><%= _('No. users joined during last year') %></h3>
+    <canvas id="yearly_users"></canvas>
+  </div>
+</div>
+<hr />
+<div class="row">
+  <div class="col-md-12">
+    <h3><%= _('Run your own filter') %></h3>
+    <% if current_user.api_token.present? %>
+      <form class="usage_index">
+        <%= hidden_field_tag('api_token', current_user.api_token) %>
+        <div class="row">
+          <div class="col-md-3">
+            <div class="form-group">
+              <%= label_tag('topic', _('Topic')) %>
+              <%= select_tag('topic', options_for_select(
+                [
+                  [_('Users'), 'users', { 'data-url': users_joined_api_v0_statistics_path }],
+                  [_('Completed Plans'), 'completed_plans', { 'data-url': '#' }]
+                ]), class: 'form-control') %>
+            </div>
+          </div>
+          <div class="col-md-2">
+            <div class="form-group">
+              <%= label_tag('start_date', _('Start date')) %>
+              <%= date_field_tag('start_date', nil, class: 'form-control') %>
+            </div>
+          </div>
+          <div class="col-md-2">
+            <div class="form-group">
+              <%= label_tag('end_date', _('End date')) %>
+              <%= date_field_tag('end_date', nil, class: 'form-control') %>
+            </div>
+          </div>
+          <% if current_user.can_super_admin? %>
+            <div class="col-md-3">
+              <div class="form-group">
+                <%= label_tag(:org_id, _('Organisation')) %>
+                <%= select_tag(:org_id, options_from_collection_for_select(orgs, :id, :name, current_user.org_id), class: 'form-control') %>
+              </div>
+            </div>
+          <% else %>
+            <%= hidden_field_tag(:org_id, current_user.org_id) %>
+          <% end %>
+          <div class="col-md-2">
+            <%= submit_tag(_('Go'), class: 'btn btn-default') %>
+          </div>
+        </div>
+      </form>
+    <% else %>
+      <p class="bg-warning">
+        <%= _('You don\'t have access to use the API. An api token is needed to generate usage statistics.') %>
```

Comment:
Add an extra sentence here to let them know how to get an API token. ping @sjDCC and @stephaniesimms for that. Maybe something like: `Contact the %{helpdesk_email} to request an API token for your organisation.`